### PR TITLE
Add course model

### PR DESCRIPTION
**Pull request by molodchenko**

### CODE REVIEW

File: config/database.yml.test [hunk 1]
```diff
@@ -1,12 +0,0 @@
-development:
```
Comment (arsenslyusarchuk) [inline config/database.yml.test:1]: Why this is still here?

Comment (molodchenko) [inline config/database.yml.test:1]: I don't have this file in my branch. I don't know why it is still here

Comment (arsenslyusarchuk) [inline config/database.yml.test:1]: Have u committed your changes?

Comment (molodchenko) [inline config/database.yml.test:1]: yes, all other changes have been applied

Comment (arsenslyusarchuk) [inline config/database.yml.test:1]: As I said yesterday u should have it. Copy it from master

Comment (kunik) [inline config/database.yml.test:1]: Guys, red means _deleted_...

Comment (kunik) [inline config/database.yml.test:1]: Ah, understood. There shouldn't be any changes to this file in the development branch.

**Pull request closed without merging**